client/components/card/style.scss

```diff
@@ -1,9 +1,9 @@
 /** @format */
 
 .woocommerce-card {
-	margin-bottom: $gap-large;
 	background: white;
 	border: 1px solid $core-grey-light-700;
+	margin-bottom: $gap-large;
```

Comment:
Out of curiosity, why move this? We don't have a set CSS property order for the project, would you want to add one?

Reply:
Moved it back.  Just habit.